src/components/Button/button.scss

```diff
@@ -17,4 +18,10 @@ $gray: hsl(234, 11%, 52%);
   &--blue{
     color: $gray;
   }
+
+  &--cross{
+    background-image: url("../../../public/images/icon-cross.svg");
+    padding: 0;
+    height: 30px;
```

Comment:
Let's move this one to its own CSS class (`&--small`), so we can only focus here to add the cross icon, and **if** needed we can reuse the small variant in another button.